assignments/hackyourtemperature/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "hackyourtemperature",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+      "start": "node server.js",
```

Formatting issue: `start` and `dev` commands are independent of `test`. So they should all be at the same tab level.
In the current formatting looks like they are nested within the command above.
```suggestion
      "test": "echo \"Error: no test specified\" && exit 1",
      "start": "node server.js",
      "dev": "nodemon server.js"
```